actors/miner/src/lib.rs

```diff
@@ -1728,6 +1728,96 @@ impl Actor {
         Ok(())
     }
 
+    fn prove_commit2(
+        rt: &impl Runtime,
+        params: ProveCommit2Params,
+    ) -> Result<(), ActorError> {
+        // XXX should requireActivationSuccess apply to precommit validation / proof correctness ?
+        // Or just data activation?
```

Comment:
Good question, I think it should apply to everything up to sector activation, essentially an all-or-nothing.